python/lsst/afw/image/SConscript

```diff
@@ -1,4 +1,19 @@
 ## -*- python -*-
+from lsst.sconsUtils import scripts
+scripts.BasicSConscript.pybind11(['image',
+				  'apCorrMap',
+				  'calib',
+				  'wcs',
+				  'tanWcs',
+				  'exposure',
+				  'exposureInfo',
+				  'mask',
+				  'maskedImage',
+                  'imageUtils',
+                  'imageSlice',
```

Comment:
Ordinarily spaces and tabs in a file will break python 3 but in this case you'd never know because scons always runs with python 2.